src/expressions/plyql.pegjs

```diff
@@ -15,7 +16,7 @@ var dateRegExp = /^\d\d\d\d-\d\d-\d\d(?:T(?:\d\d)?(?::\d\d)?(?::\d\d)?(?:.\d\d\d
 
 // See here: https://www.drupal.org/node/141051
 var reservedWords = {
-  ALL: 1, AND: 1, AS: 1, ASC: 1,
+  ABS: 1, ABSOLUTE: 1, ALL: 1, AND: 1, AS: 1, ASC: 1,
```

Comment:
You do not need to add there as it turns out (I only realized my mistake a few days ago).

Check out: https://dev.mysql.com/doc/refman/5.5/en/keywords.html

None of the new actions you are adding are reserved. You can double check by running `SELECT 1 AS ABS` it works fine for me.